whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/FullTextIndex.java

```diff
@@ -307,7 +307,31 @@ public RpslObject filterRpslObject(final RpslObject rpslObject) {
 
         return new RpslObject(rpslObject.getObjectId(), attributes);
     }
-    
+
+    public List<RpslAttribute> filterRpslAttributes(final Set<AttributeType> attributeType, final Map<String, Object> hitAttributes) {
+
+        List<RpslAttribute> attributes = Lists.newArrayList();
```

Comment:
FulltextIndex is also being used by lucene . If its specific to elastic can you shift this code in that class itself and remove the reference to fulltextIndex?